From f59b0ea30575d8cfe02e9fbe039ce35e66fae716 Mon Sep 17 00:00:00 2001
From: Ben Hutchings <ben@decadent.org.uk>
Date: Tue, 6 Jul 2010 01:42:07 +0100
Subject: [PATCH] Partially revert "CPT: Replace legacy net statistics with netns one"

This reverts the structure change in commit
7e99ed1bc34b60ed42eb2008edbb4f98684edb0a.

Signed-off-by: Ben Hutchings <ben@decadent.org.uk>
---
 include/linux/ve.h |    8 ++++++++
 1 files changed, 8 insertions(+), 0 deletions(-)

diff --git a/include/linux/ve.h b/include/linux/ve.h
index e473727..e0e045a 100644
--- a/include/linux/ve.h
+++ b/include/linux/ve.h
@@ -239,6 +239,14 @@ struct ve_struct {
 	struct ve_ipt_recent	*_ipt_recent;
 	struct ve_xt_hashlimit	*_xt_hashlimit;
 #endif /* CONFIG_VE_IPTABLES */
+
+#if defined(CONFIG_IPV6) || defined(CONFIG_IPV6_MODULE)
+	struct ipstats_mib	*_ipv6_statistics[2];
+	struct icmpv6_mib	*_icmpv6_statistics[2];
+	struct icmpv6msg_mib	*_icmpv6msg_statistics[2];
+	struct udp_mib		*_udp_stats_in6[2];
+	struct udp_mib		*_udplite_stats_in6[2];
+#endif
 #endif
 	wait_queue_head_t	*_log_wait;
 	unsigned		*_log_start;
-- 
1.7.1

